server/src/test/java/org/apache/iotdb/db/integration/IoTDBRestartIT.java

```diff
@@ -113,6 +114,11 @@ public void testRestartDelete()
 
     try {
       EnvironmentUtils.restartDaemon();
+      StorageEngine.getInstance().recover();
+      // wait for recover
+      while(!StorageEngine.getInstance().isAllSgReady()){
+        Thread.sleep(500);
```

Comment:
Better to add some logs if it has waited for too long, so we can know what went wrong.

Reply:
fixed~